README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# Ansible Network Collection for Common Code (netcommon)
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/builds?project=ansible-collections%2Fansible.netcommon) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/ansible.netcommon)](https://codecov.io/gh/ansible-collections/ansible.netcommon)
+
+The Ansible ``ansible.netcommon`` collection includes common network content to help automate the management of network devices.
+This includes  connection plugins, such as ``httpapi`` and ``netconf``.
+
+## Included content
+
+This collection includes:
+- Common network modules, such as ``net_*``, ``cli_*``, ``netconf_*``, and ``restconf_*`` modules.
+- Common connection plugins, such ``httpapi`` and ``network_cli``.
+- ``ipadrr`` and ``network`` filters.
+
+Click the ``Content`` button to see the full list of content included in this collection.
+
+## Installing this collection
+
+You can install the ``ansible.netcommon`` collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install ansible.netcommon
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: ansible.netcommon
+    version: 0.0.3
+```
+## Using this collection
+
+The most common use case for this collection is to include it as a dependency in a network device-specific collection. See the [Vyos collection](https://github.com/ansible-collections/vyos) for an example of this.
```

Comment:
Maybe more information is required on how to use the plugins in these collections. Like using the fully qualified collection name for the connections for eg. `ansible.netcommon.network_cli`